services/kommander/0.14.0/defaults/cm.yaml

```diff
@@ -102,6 +102,7 @@ data:
     - "dex"
     - "dex-k8s-authenticator"
     - "nkp-insights-management"
+    - "nkp-pulse-management"
```

Comment:
Note to the reviewer, this adds `nkp-pulse-management` to the list of applications that get migrated during the Essentials -> Enterprise migration between two management clusters.

This *does not mean* that nkp-pulse-management is automatically enabled on the management cluster.

Reply:
Apparently leaving comments blocks merging, which I didn't know. Resolving.

Reply:
It adds `nkp-pulse-management` to the list of AppDeployments which are **not** copied from formerly self-attached cluster to the new management cluster by the Expansion controller.
The outcome is that `nkp-pulse-management` is removed from the adopted cluster, similarly to Kommander components and `nkp-insights-management`.